free-1.md

```diff
@@ -45,6 +78,14 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I was somewhat inspired by [Ari's CS111 project from
+2014](https://github.com/AriHC/RuleBook), but our ideas are different in that
+his scope was more limited to tic-tac-toe-like games and automatically made a
+playable computer game, while mine will accept more general games and only
+output rules. I also found some other cool game generators like
+[PuzzleScript](http://www.puzzlescript.net/editor.html). But what makes my
+project different is that it can specify arbitrary games, and trades this
+flexibility for the lack of actually implementing them.
```

Comment:
This may be a hard tradeoff to make.  I think it might even be possible to prove some form of equivalence.  See comment on L110.